portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,216 @@
+  <div class="mySlides fade">
+    <div class="numbertext">1 / 10</div>
+    <img src="image/shoreacres.jpg" style="width:100%">
+    <div class="text">Where I grew up</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">2 / 10</div>
+    <img src="image/ceasari.png" style="width:100%">
+    <div class="text">My first adventure while working in Israel</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">3 / 10</div>
+    <img src="image/capetown.jpg" style="width:100%">
+    <div class="text">My favorite photo, from Cape Town taken on Table Mountain</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">4 / 10</div>
+    <img src="image/jerusalem.jpg" style="width:100%">
+    <div class="text">My trip to Jerusalem</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">5 / 10</div>
+    <img src="image/thailand.png" style="width:100%">
+    <div class="text">Kanchanaburi Thailand, the River Kwai</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">6 / 10</div>
+    <img src="image/bigbuddha.png" style="width:100%">
+    <div class="text">The Big Buddha at Ngong Ping</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">7 / 10</div>
+    <img src="image/hkfishingtown.jpg" style="width:100%">
+    <div class="text">A small fishing town at the base of Ngong Ping</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">8 / 10</div>
+    <img src="image/canada.jpg" style="width:100%">
+    <div class="text">Canada heading to the mine</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">9 / 10</div>
+    <img src="image/govball.png" style="width:100%">
+    <div class="text">Governer's Ball Music Festival</div>
+  </div>
+
+  <div class="mySlides fade">
+    <div class="numbertext">10 / 10</div>
+    <img src="image/towerbridge.jpg" style="width:100%">
+    <div class="text">Tower Bridge with my Girlfriend Ashley, on our first trip to London</div>
+  </div>
+
+  <!-- Next and previous buttons -->
+  <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+  <a class="next" onclick="plusSlides(1)">&#10095;</a>
+</div>
+<br>
+
+<!-- The dots/circles -->
+<div style="text-align:center">
+  <span class="dot" onclick="currentSlide(1)"></span>
+  <span class="dot" onclick="currentSlide(2)"></span>
+  <span class="dot" onclick="currentSlide(3)"></span>
+  <span class="dot" onclick="currentSlide(4)"></span>
+  <span class="dot" onclick="currentSlide(5)"></span>
+  <span class="dot" onclick="currentSlide(6)"></span>
+  <span class="dot" onclick="currentSlide(7)"></span>
+  <span class="dot" onclick="currentSlide(8)"></span>
+  <span class="dot" onclick="currentSlide(9)"></span>
+  <span class="dot" onclick="currentSlide(10)"></span>
+
+    <script>
+let slideIndex = 0;
+showSlidesAuto();
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
```

Comment:
took this suggestion, honestly I realized in my programming languages this would throw an error, but copied a lot of this code from a tutorial, and figured this is just people using js's weirdness to get rid of a line of code, and it's still clear that an assignment is happening so I thought I'd leave it.